doc/source/release/1.14.0-notes.rst

```diff
@@ -49,49 +63,69 @@ New features
 
 `scipy.optimize` improvements
 =============================
+- `scipy.optimize.HessianUpdateStrategy` now also accepts square arrays for
+  ``init_scale``.
+- A new method, ``cobyqa``, has been added to `scipy.optimize.minimize` - this
+  is an interface for COBYQA (Constrained Optimization BY Quadratic
+  Approximations), a derivative-free optimization solver, designed to
+  supersede COBYLA, developed by the Department of Applied Mathematics, The
+  Hong Kong Polytechnic University.
+- There are some performance improvements in
+  `scipy.optimize.differential_evolution`.
+- `scipy.optimize.approx_fprime` now has subquadratic space complexity.
 
 
 `scipy.signal` improvements
 ===========================
+- `scipy.signal.minimum_phase` has a new argument ``half``, allowing the
+  provision of a filter of the same length as the linear-phase FIR filter
+  coefficients and with the same magnitude spectrum.
 
 
 `scipy.sparse` improvements
 ===========================
-
+- A special case has been added to handle multiplying a ``dia_array`` by a
+  scalar, which avoids a potentially costly conversion to CSR format.
+- `scipy.sparse.csgraph.yen` has been added, allowing usage of Yen's K-Shortest
+  Paths algorithm on a directed on undirected graph.
 
 
 `scipy.spatial` improvements
 ============================
-
+- ``Rotation`` supports an alternative "scalar-first" convention of quaternion
+  component ordering. It is available via the keyword argument ``scalar_first``
+  of ``from_quat`` and ``as_quat`` methods.
+- Some minor performance improvements for inverting of ``Rotation`` objects.
 
 `scipy.special` improvements
 ============================
+- Added `scipy.special.log_wright_bessel`, for calculation of the logarithm of
+  Wright's Bessel function.
+- Generalized ufunc support has been expanded.
+- The relative error in `scipy.special.hyp2f1` calculations has improved
+  substantially.
+- Improved behavior of ``boxcox``, ``inv_boxcox``, ``boxcox1p``, and
+  ``inv_boxcox1p`` by preventing premature overflow.
```

Comment:
Why is this an enhancement instead of a bug fix?

Reply:
It was one of those cases where the function was ok up to 1e300 but now it's good up to 1e308 by adding a special case. I wouldn't consider the original implementation to have a bug even though it didn't work as well as it could have for all possible inputs; there was just room for improvement.